pre-reg-113-115-application-default.properties

```diff
@@ -0,0 +1,86 @@
+#helm chart properties
+#db.source.dbuser.password
+#db.destination.dbuser.password
+#server.port
+#mosip.base.url
+#mosip.prereg.client.secret
+#encryptBaseUrl
+#decryptBaseUrl
+#source.object.store.secretkey
+#destination.object.store.secretkey
+#isNewDatabase=true
```

Comment:
We don't need the above properties ? Why commented ?

Reply:
these properties I thought of adding in helm chart as environment variables.